docs/backup-restore.md

```diff
@@ -127,6 +133,12 @@ You must enable the PITR feature for DynamoDB tables. If you're using [ScalarDB
 
 To specify a transactionally consistent restore point, pause your application that is using ScalarDB with DynamoDB as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
 </div>
+<div id="YugabyteDB_Managed2" class="tabcontent" markdown="1">
+
+Clusters are backed up automatically based on the backup policy, and these backups are retained for a specific duration. You can also perform on-demand backups. For details on performing backups, see [YugabyteDB Managed: Back up and restore clusters](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-clusters/backup-clusters/).
```

Comment:
Sorry, I meant this paragraph should probably be removed because it sounds like automatic backups based on the policy are just fine.
What do you think?
(I came up with the next paragraph to emphasize taking backups should be done during a paused duration.)

Reply:
Oh, sorry, I misread your comment and just merged the change w/o removing the first paragraph.

I thought the first paragraph was okay since it described what the backups on YugabyteDB Managed are like, and then the second paragraph explained that pausing was necessary in that case. But, I agree that users who read only the first paragraph might think no pause is needed. I'll remove it.

Reply:
@feeblefakie Addressed it in https://github.com/scalar-labs/scalardb/pull/1710/commits/92098ba2f99a3ba4daee59bb057e40eb72416ad4. Thanks for the great feedbacks!